urdfenvs/iris/resources/iris.py

```diff
@@ -0,0 +1,38 @@
+import os
+import numpy as np
+
+from urdfenvs.urdfCommon.quadrotor import QuadrotorModel
+
+
+class IRIS(QuadrotorModel):
+    """IRIS model
+
+    Attributes
+    ---------
+    _scaling: float
+        The size scaling in which the urdf should be spawned.
+        This also effects the dynamics of the system.
+
+    """
+
+    def __init__(self):
+        n = 3
```

Comment:
Why `n=3`? Wouldn't `n=4` be better justified because of the number of rotors?

Reply:
I think `n` is the dimension of the space so I set `n=3`. I didn't check the code in `GenericRobot` class therefore I don't know the consequences if I set `n=4`.

Reply:
Although drone has 4 rotors it's still a 6-DoF system, so I think it's better to set `n=6` according to the meaning of `n` in GenericRobot class?